.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Añadí el directorio de .idea/

Reply:
slash is not necessary.  
place comments in Discussions and close them if you did what was asked for.